fe/fe-core/src/main/java/org/apache/doris/service/FrontendOptions.java

```diff
@@ -98,6 +98,10 @@ public static String getHostname() {
         return localAddr.getHostName();
     }
 
+    public static String getCanonicalHostName() {
```

Comment:
I put the hostname and IP in the etc/hosts file, but it can't be resolved，
when this method can get HostName?

Reply:
Are you running on Docker or physical machines?  
This issue may not be related to gethostname or getCanonicalHostName.